packages/nimbus/src/components/list/list.mdx

```diff
@@ -17,4 +17,296 @@ tags:
+      <List.Item>
+        <List.Indicator>
+          <Icon color="critical.11"><Icons.Close /></Icon>
+        </List.Indicator>
+        Not eating a treat
+      </List.Item>
+    </List.Root>
+  </Stack>
+)
+```
+
+#### Alignment
+
+List items are aligned to the left, center and the right. Choose what suits the information to be shown based on language and data.
+
+```jsx-live
+const App = () => (
+  <Stack direction="horizontal" gap="800" alignItems="flex-start">
+      <Stack gap="200">
+      <Text fontWeight="bold" textAlign="left">Left</Text>
+    <List.Root asChild>
+      <ul>
+        <List.Item textAlign="left">Red pandas</List.Item>
+        <List.Item textAlign="left">Pangolins</List.Item>
+        <List.Item textAlign="left">Opossums</List.Item>
+        <List.Item textAlign="left">Aye-ayes</List.Item>
+        <List.Item textAlign="left">Okapi</List.Item>
+      </ul>
+    </List.Root>
+        </Stack>
+        <Stack gap="200">
+      <Text fontWeight="bold" textAlign="center">Center</Text>
+    <List.Root asChild>
+      <ul>
+        <List.Item textAlign="center">Red pandas</List.Item>
+        <List.Item textAlign="center">Pangolins</List.Item>
+        <List.Item textAlign="center">Opossums</List.Item>
+        <List.Item textAlign="center">Aye-ayes</List.Item>
+        <List.Item textAlign="center">Okapi</List.Item>
+      </ul>
+    </List.Root>
+    </Stack>
+    <Stack gap="200">
+      <Text fontWeight="bold" textAlign="right">Right</Text>
+     <List.Root asChild>
+      <ul>
+        <List.Item textAlign="right">Red pandas</List.Item>
+        <List.Item textAlign="right">Pangolins</List.Item>
+        <List.Item textAlign="right">Opossums</List.Item>
+        <List.Item textAlign="right">Aye-ayes</List.Item>
+        <List.Item textAlign="right">Okapi</List.Item>
+      </ul>
+    </List.Root>
+    </Stack>
+  </Stack>
+)
+```
+
+
+#### Nested items, added gap space, and item wrapping
+When nesting related items, maintain proper spacing and wrapping to keep the structure intact. Avoid long lines of text; if necessary, rephrase to make it shorter or use body text instead of bullet points, which should be reserved for concise information.
+
+```jsx-live
+const App = () => (
+  <Stack direction="horizontal" gap="800" alignItems="flex-start">
+    <Stack gap="200">
+      <Text fontWeight="bold">Nested items</Text>
+      <List.Root asChild>
+        <ul>
+          <List.Item>
+            Password requirements:
+            <List.Root asChild pl="400">
+              <ul>
+                <List.Item>8 characters and 3 numbers</List.Item>
+                <List.Item>One special character (!@#$%^&*)</List.Item>
+              </ul>
+            </List.Root>
+          </List.Item>
+          <List.Item>
+            Name and email requirements
+            <List.Root asChild pl="400">
+              <ul>
+                <List.Item>Validation process</List.Item>
+              </ul>
+            </List.Root>
```

Comment:
I struggled to get the `Nested items` stack to format correctly w/ in this example. It's _identical_ to the Accessibility code example below. 
<img width="631" height="292" alt="Screenshot 2025-11-11 at 14 39 01" src="https://github.com/user-attachments/assets/9486d6fe-375f-4eef-b6b8-6ee787892290" />

suggestions welcome. it's supposed to look slick like :
<img width="322" height="113" alt="Screenshot 2025-11-11 at 14 40 20" src="https://github.com/user-attachments/assets/625fb32d-f099-47ee-9fc3-4b7789cc4c7d" />